packages/web-components/src/components/rux-input/test/rux-input.spec.tsx

```diff
@@ -16,7 +16,7 @@ describe('rux-input', () => {
                   <slot name="label"></slot>
                 </span>
               </label>
-              <input aria-invalid="false" autocomplete="on" class="rux-input rux-input--medium" id="rux-input-1" type="text" value="">
```

Comment:
@markacianfrani if you're wondering why this was removed, it shouldn't have been there in the first place. The logic before would change autocomplete to on if the type wasn't password, it now is not there if it's not set and is `off` if it's a pw type.